libs/qemu-docker/windows/src/vm/setup/setup.ps1

```
@@ -34,6 +34,73 @@ Write-Host "============================================"
+# Remove common startup entries that can show windows on login
+# (OneDrive, Microsoft Teams, Windows Security, etc.)
+$startupKeys = @(
+    'HKCU:\Software\Microsoft\Windows\CurrentVersion\Run',
+    'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Run'
+)
+$startupEntriesToRemove = @('OneDrive', 'MicrosoftTeams', 'Teams', 'SecurityHealth')
+foreach ($key in $startupKeys) {
+    foreach ($entry in $startupEntriesToRemove) {
+        try {
+            Remove-ItemProperty -Path $key -Name $entry -ErrorAction SilentlyContinue
+        } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)